packages/theming/src/utils/getColor.ts

```diff
@@ -316,10 +316,7 @@ export const getColor = memoize(
 
       _hue = value.hue;
       _shade = value.shade;
-
-      if (value.transparency !== undefined) {
-        _transparency = value.transparency;
-      }
+      _transparency = _transparency === undefined ? value.transparency : _transparency;
```

Comment:
If `transparency` exists along with a `variable` parameter, the `transparency` should override the variables `rgba` alpha value.